xonsh/environ.py

```diff
@@ -242,24 +243,6 @@ def detype(self):
         self._detyped = ctx
         return ctx
 
-    def replace_env(self):
-        """Replaces the contents of os.environ with a detyped version
-        of the xonsh environement.
-        """
-        if self._orig_env is None:
-            self._orig_env = dict(os.environ)
-        os.environ.clear()
-        os.environ.update(self.detype())
-
-    def undo_replace_env(self):
-        """Replaces the contents of os.environ with a detyped version
-        of the xonsh environement.
-        """
-        if self._orig_env is not None:
-            os.environ.clear()
-            os.environ.update(self._orig_env)
-            self._orig_env = None
```

Comment:
I think that these are still potentially useful to users.  Do you mind putting them back in?